flink-runtime/src/test/java/org/apache/flink/runtime/dispatcher/DispatcherTest.java

```diff
@@ -219,11 +156,19 @@ private TestingDispatcher createAndStartDispatcher(
             TestingHighAvailabilityServices haServices,
             JobManagerRunnerFactory jobManagerRunnerFactory)
             throws Exception {
+        JobGraphWriter jobGraphWriter;
+        try {
+            jobGraphWriter = haServices.getJobGraphStore();
+        } catch (IllegalStateException e) {
+            jobGraphWriter = NoOpJobGraphWriter.INSTANCE;
+        }
```

Comment:
Instead of doing this, would it make sense to initialize the `haServices` field with a `StandaloneJobGraphStore`?

Reply:
👍